cmd/notation/plugin.go

```diff
@@ -62,3 +108,169 @@ func listPlugins(command *cobra.Command) error {
 	}
 	return tw.Flush()
 }
+
+func installPlugin(command *cobra.Command, args []string, force bool) error {
+	if len(args) != 1 {
+		return errors.New("missing plugin package")
+	}
+
+	pluginPath := args[0]
```

Comment:
We should check length of `args` and assign `args[0]` in `&cobra.command{Args:xxx}`.

Reply:
Do you have an example? I'm not sure how to implement it. Thanks @patrickzheng200 :)